eng/native/configuretools.cmake

```diff
@@ -55,11 +56,10 @@ if(NOT WIN32 AND NOT CLR_CMAKE_TARGET_BROWSER)
     elseif(CMAKE_CROSSCOMPILING AND NOT DEFINED CLR_CROSS_COMPONENTS_BUILD AND
         CMAKE_SYSTEM_PROCESSOR MATCHES "^(armv8l|armv7l|armv6l|aarch64|arm|s390x|ppc64le)$")
       set(TOOLSET_PREFIX "${TOOLCHAIN}-")
-    else()
+    elseif(CLR_CMAKE_TARGET_ALPINE_LINUX AND CMAKE_C_COMPILER_VERSION VERSION_LESS 8.0)
```

Comment:
Do we still care about old clang versions?

Reply:
Good point, we don't. We currently build on Alpine 3.13, which has llvm 10.